pepper-apis/src/test/java/org/broadinstitute/ddp/route/UserActivityInstanceListRouteStandaloneTest.java

```diff
@@ -62,7 +63,7 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class UserActivityInstanceListRouteTest extends IntegrationTestSuite.TestCase {
+public class UserActivityInstanceListRouteStandaloneTest extends IntegrationTestSuite.TestCaseWithCacheEnabled {
```

Comment:
The tests were failing because some of the tests changes the activity definitions mid-flight and need to clear the `ActivityDefStore` cache. But server was running in a different process in CircleCI, so I moved this to be a "standalone" test.